openapi.yaml

```
@@ -2453,18 +2485,18 @@ paths:
                 type: object
                 properties:
-                  data:
+                  status:
                     type: string
                   processing_time:
                     type: number
                   processing_id:
                     type: string
-                  processing_count:
-                    type: integer
+                  messsage: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)